src/rsz/src/RepairSetup.cc

```diff
@@ -343,10 +345,11 @@ void RepairSetup::repairSetup(const Pin* end_pin)
 }
 
 /* This is the main routine for repairing setup violations. We have
- - upsize driver (step 1)
- - rebuffer (step 2)
- - swap pin (step 3)
- - split loads
+ - remove driver (step 1)
```

Comment:
remove buffer?  Removing the driver sounds wrong.

Reply:
Buffer can be removed either at driver or load.   Setup fixing transforms are geared towards driver changes, so I can add driver removal to the list of transforms.